File: GO.ejercicios/bucles.for.go
package main

import "fmt"

func main() {
    // Crear la lista de nombres
    nombres := []string{"Esteban", "Hans", "Jhon", "Juan Pablo"}

    // Imprimir los nombres
    for _, nombre := range nombres {
        fmt.Println(nombre)
    }

    // Crear la lista de personas (como slice de mapas)
    personas := []map[string]string{
        {"nombre": "Esteban", "Edad": "28"},
        {"nombre": "Hans", "Edad": "27"},
        {"nombre": "Jhon", "Edad": "41"},
        {"nombre": "Juan Pablo", "Edad": "23"},
    }

    // Imprimir los nombres y edades
    for _, persona := range personas {
        fmt.Println(persona["nombre"], persona["Edad"])
    }
}

File: RUBY.EJERCICIOS/BUCLES.for.rb
# Crear la lista de nombres
nombres = ["Esteban", "Hans", "Jhon", "Juan Pablo"]

# Imprimir los nombres
nombres.each do |nombre|
  puts nombre
end

# Crear la lista de personas (como lista de Hashes)
personas = []

# Crear los Hashes para cada persona
a = { "nombre" => "Esteban", "Edad" => 28 }
b = { "nombre" => "Hans", "Edad" => 27 }
c = { "nombre" => "Jhon", "Edad" => 41 }
d = { "nombre" => "Juan Pablo", "Edad" => 23 }

# Agregar los Hashes a la lista
personas.push(a, b, c, d)

# Imprimir los nombres y edades
personas.each do |persona|
  puts "#{persona["nombre"]} #{persona["Edad"]}"
end